### PR TITLE
Sous Ownership

### PR DESCRIPTION
**Pull request by Xorima**
Update to circleci
Fix metadata
updated changelog
Depreciated support for sous-unsupported os (fedora, smartos)

### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @chef-cookbooks/cookbook_engineering_team
```
Comment (tas50) [inline .github/CODEOWNERS]: hehe

File: kitchen.yml [hunk 2]
```diff
@@ -20,7 +20,6 @@ platforms:
   - name: freebsd-10
```
Comment (tas50) [inline kitchen.yml]: FreeBSD 10 is EOL so this can go away

File: README.md [hunk 3]
```diff
@@ -11,13 +11,12 @@ Installs and configures Squid as a caching proxy.
 - Debian 7+
 - Ubuntu 12.04+
 - RHEL/CentOS/Amazon/Scientific 6+
-- Fedora
 - openSUSE / openSUSE Leap
 - FreeBSD 10+
```
Comment (tas50) [inline README.md]: This should be FreeBSD 11+

File: kitchen.yml (hunk 2 above)
Comment (Xorima) [reply kitchen.yml]: YAY!!!!

File: .github/CODEOWNERS (hunk 1 above)
Comment (Xorima) [reply .github/CODEOWNERS]: Should we just set it to be * tas50 ;)

File: .kitchen.yml [hunk 4]
```diff
@@ -17,10 +17,8 @@ platforms:
   - name: debian-8
   - name: debian-9
   - name: fedora-28
-  - name: freebsd-10
   - name: freebsd-11
```
Comment (damacus) [inline .kitchen.yml]: Nope nope please 😂

Comment (Xorima) [reply .kitchen.yml]: blame @tas50 he had it before? 

It's passing/working now, do we want to support it till it's too much pain, or drop support today?

**Pull request merged**